src/backends/fixerio.ts

```diff
@@ -0,0 +1,134 @@
+import fetchUri from 'node-fetch'
+import BigNumber from 'bignumber.js'
+import { AccountInfo } from '../types/accounts'
+import { IBackend } from '../types/backend'
+
+import { create as createLogger } from '../common/log'
+const log = createLogger('fixerio')
+
+const RATES_API = 'https://api.fixer.io/latest'
+
+export interface FixerIoOptions {
+  spread: number,
+  ratesApiUrl: string,
+  getInfo: (accountId: string) => AccountInfo,
+  getAssetCode: (accountId: string) => string
+}
+
+/**
+ * Dummy backend that uses Fixer.io API for FX rates
+ */
+export default class FixerIoBackend implements IBackend {
+  protected spread: number
+  protected getInfo: (accountId: string) => AccountInfo
+  protected getAssetCode: (accountId: string) => string
+
+  protected rates: {
+    [key: string]: number
+  }
+  protected currencies: string[]
+
+  /**
+   * Constructor.
+   *
+   * @param {Integer} opts.spread The spread we will use to mark up the FX rates
+   * @param {String} opts.ratesApiUrl The API endpoint we will request rates from
```

Comment:
The jsdoc is out of date. We should either update it or remove it if it's no longer necessary (since we're switching to typescript)

Reply:
Updated to tsdoc in next commit.